### DISCUSSION TITLE
Blazor Web (Auto)

### DISCUSSION THREAD

**Discussion by FBODeveloper** [balta-io/forum#433] [Tira Dúvidas] [answered]
Para desenvolver um app que seja client/server ( não seria um aplicativo web em si) usar o Blazor Web como Auto  daria pra criar sem problemas? Dessa forma nas maquinas clientes só rodaria o client ou toda aplicação teria que estar na máquina cliente?
Desculpem se a pergunta é meio confusa ou simplista demais.... estou começando... valeu

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @FBODeveloper, como vai?
Poderia sim... neste caso a aplicação fica no servidor. Se for somente WASM aí sim ela pode ficar totalmente na máquina do cliente (ou também no servidor).
